package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.26.0",
+    "hint": "^7.1.1",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  }
+}
```

Comment:
- [ ] Your Package.json is missing the script `npm start`, thereby preventing your project from running locally on the server. You will need to set up webpack to have the` start` or run` script `command in my opinion 😄 . Kindly set up webpack properly so that your app can run properly 👍🏼

Reply:
@vickymarz But i have used this package in all my previous projects and it is working on my local machine.

Reply:
I would recommend you use webpack for this project. Since you have been taught, it will be advisable to implement your javascript project with webpack. 

You will have something like this 👇🏼 if you properly set up webpack. Please consider using it 👍🏼 
![image](https://user-images.githubusercontent.com/63473584/183765335-8c2a6f70-2da4-4d29-984d-de795156e9b2.png)